test/server_test.py

```diff
@@ -46,28 +33,47 @@ def serverTests():
 	test_pass = 'testpass'
 	test_code = 'Test1234'
 	test_email = 'test@email.com'
-	salt = 'saltything'
-	test_user = {
-		'name': test_name,
-		'pass_salt': salt,
-		'pass_hash': scrypt.hash(test_pass, salt)
-	}
+	test_salt = 'saltything'
```

Comment:
I think this string should be 'pickle'. I have seen test_salt here a few times and thought this. I think it's time to put my foot down.